assets/javascripts/modules/analytics/omniture.js

```diff
@@ -18,73 +17,72 @@ define([
 
     function trackLink(link) {
         omniture = omniture || init();
-        omniture.then(function(){
+        omniture.then(function () {
             s.tl(true, 'o', link);
         });
     }
 
-    function trackEvent(prop17, pageName, products) {
-        omniture = omniture || init();
-
-        omniture.then(function(){
-            if (prop17) {
-                s.prop17 = prop17;
-                s.pageName = pageName;
-                s.products = products;
-                s.t();
-            }
-        });
-    }
-
-    function onSuccess() {
-        var s_code;
-
-        window.s_account = 'guardiangu-subscribe,guardiangu-network';
-        s = s_gi('guardiangu-network');
+    function setTracker(s) {
+        var pageInfo = guardian.pageInfo,
+            productData = guardian.productData;
```

Comment:
I'd prefer to scope this to a single key, so `slug, pageInfo , productData` under one key.